src/index.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import pick from 'lodash/fp/pick';
+import omitF from 'lodash/fp/omit';
 import compact from 'lodash/compact';
 import flowRight from 'lodash/flowRight';
+import isUndefined from 'lodash/isUndefined';
 import { t, props } from 'tcomb-react';
 import { skinnable, pure, contains } from 'revenge';
 import _declareConnect from 'buildo-state/lib/connect';
 import _declareQueries from 'react-avenger/lib/queries';
 import _declareCommands from 'react-avenger/lib/commands';
-import noLoaderLoading from './noLoaderLoading';
+import { defaultIsReady, noLoaderLoading } from 'react-avenger/lib/loading';
 import displayName from './displayName';
 
+const stripUndef = omitF(isUndefined);
```

Comment:
Oh, btw I just discovered that there was a bug here. I needed `omitBy` from `lodash/fp/omitBy`